pkgs/games/0ad/game.nix

```diff
@@ -9,8 +9,8 @@
   fmt,
   libidn,
   pkg-config,
-  spidermonkey_78,
```

Comment:
thank god, we can now get rid of another python 3.9 user. Going to open a PR for that after merging this.